actions/workflows/st2_prep_release.yaml

```diff
@@ -48,6 +48,28 @@ st2cd.st2_prep_release:
                 local_repo: <% 'st2_' + $.local_repo_sfx %>
                 hosts: <% $.host %>
                 cwd: <% $.cwd %>
+            on-success:
+                - prep_st2_dockerfiles
+        prep_st2_dockerfiles:
```

Comment:
It would be probably much better both logically and technically to move these new steps at the end of the workflow, after preparing all other st2 components like st2web, st2chatops, st2-packages.